docs/getting-started/introduction.md

```diff
@@ -18,8 +18,8 @@ developers and an intuitive App for non-technical users.** Written entirely in J
 [Node.js](https://nodejs.dev) and [Vue.js](https://vuejs.org)), Directus is completely open-source, modular, and
 extensible, allowing it to be fully tailored to the requirements of any project.
 
-The platform can be used as a headless content management, or for democratizing data (customer, inventory, business, 
-project, etc) across an organization.
+The platform can be used for an individual purpose, such as headless content management, or more broadly to democratize an entire organization's data (customers, inventory, business intelligence, 
```

Comment:
@benhaynes "as a headless content management" missing "system"? 🤣

Reply:
I was trying to avoid being too CMS, this is describing a _purpose_, ""headless content management", so I think it works. Do you think it's clearer to rephrase as a CMS?

Reply:
Ahh gotcha. Misread that; I thought this was reverted back to the previous PR